docs/design/2022-03-10-backup-with-batch-create-table.md

```diff
@@ -0,0 +1,117 @@
+# TiDB Design Documents
+
+- Author(s): [fengou1](http://github.com/your-github-id)
+- Discussion PR: https://github.com/pingcap/tidb/pull/27036
+- Tracking Issue: https://github.com/pingcap/tidb/issues/28763
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Motivation or Background](#motivation-or-background)
+* [Detailed Design](#detailed-design)
+* [Test Design](#test-design)
+ * [Functional Tests](#functional-tests)
+ * [Compatibility Tests](#compatibility-tests)
+ * [Benchmark Tests](#benchmark-tests)
+
+## Introduction
+
+The proposal of this design is to speed up the BR restore.
+
+## Motivation or Background
+
+The cluster has 6 TiB data, 30k tables, and 11 TiKVs. When I use BR to backup and restore the cluster, The speed is particularly slow. After investigation, BR can only create 2 tables per second, the entire store speed takes nearly 4h, and the execution time spent creating tables is close to 4h. It can be seen that the execution speed of ddl is the bottleneck in this scenario.
+
+## Detailed Design
+
+Backup and Restore for massive tables is extremely slow, the bottleneck of creating the table is to waiting for the schema version change. Each table creates ddl cause a schema version change. 60000 tables have almost 60000 times schema change, it is very cost of restoring massive table
+
+Currently, BR uses an internal interface named CreateTableWithInfo to create table, which creating the table and wait the schema changing one-by-one, omitting the sync of the ddl job between BR and leader, the procedure of creating one table would be like this:
+
+for _, t := range tables {
+ RunInTxn(func(txn) {
+ m := meta.New(txn)
+ schemaVesrion := m.CreateTable(t)
+ m.UpdateSchema(schemaVersion)
+ })
+ waitSchemaToSync() // <- This would notify and then 
+ // waiting for all other TiDB nodes are synced with the latest schema version. 
+}
+
+the new design will introduce a new batch create table api
```

Comment:
Introduce more about the logic and the api interface. You don't need to be so detailed in the proposal.